From 2459aac0d32969a889eaa1f5dc9db524422194e0 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 16:13:40 -0400
Subject: [ethernet] be2net: update driver version to 10.2

Message-id: <1410192881-30702-18-git-send-email-ivecera@redhat.com>
Patchwork-id: 92410
O-Subject: [RHEL7.1 PATCH 17/78] be2net: update driver version to 10.2
Bugzilla: 1088575
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>

BZ: #1088575

Upstream commit(s):
commit d52afde96ffda8c9d3f0e32ffe87b6cc3290580e
Author: Sathya Perla <sathya.perla@emulex.com>
Date:   Tue Mar 11 18:53:09 2014 +0530

    be2net: update driver version to 10.2

    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be.h b/drivers/net/ethernet/emulex/benet/be.h
index a1ded87..725715d 100644
--- a/drivers/net/ethernet/emulex/benet/be.h
+++ b/drivers/net/ethernet/emulex/benet/be.h
@@ -34,7 +34,7 @@
 #include "be_hw.h"
 #include "be_roce.h"
 
-#define DRV_VER   "10.0.600.0r"
+#define DRV_VER   "10.2r"
 #define DRV_NAME  "be2net"
 #define BE_NAME   "Emulex BladeEngine2"
 #define BE3_NAME  "Emulex BladeEngine3"
-- 
1.7.1